config.json

```diff
@@ -13,6 +14,5 @@
 	"Local": {
 		"BucketPort": 8333,
 		"Driver": "docker-desktop"
-	},
-	"ZetaforgeIsDev": true
```

Comment:
If this var is removed it needs to be updated in the analytics code as it's used for checking if it's a dev version or prod version

Reply:
I simply renamed it to IsDev to be consistent with existing variable names

Reply:
Other code refers to `ZetaforgeIsDev` so those sections need to be updated to read the new var name

Reply:
I did not find any reference of `ZetarforgeIsDev` in the go code at least.